src/easylink/graph_components.py

```diff
@@ -190,14 +190,14 @@ class ImplementationGraph(nx.MultiDiGraph):
     Notes
     -----
     An ``ImplementationGraph`` is a low-level abstraction; it represents the *actual
-    implementations* of each :class:`~easylink.step.Step` in the pipeline. This
-    is in contrast to a :class:`StepGraph`, which can be an intricate nested structure
+    implementations* of each :class:`~easylink.step.Step` in a pipeline. This is
+    in contrast to a :class:`StepGraph`, which can be an intricate nested structure
     due to the various complex and self-similar ``Step`` instances (which represent
     abstract operations such as "loop this step N times"). An ``ImplementationGraph``
-    is the flattened and concrete graph of ``Implementations`` to run.
+    is the flattened and concrete graph of ``Implementations`` in a given pipeline.
 
-    The highest level ``ImplementationGraph`` is the that of the entire
-    :class:`~easylink.pipeline_graph.PipelineGraph`.
+    The highest level ``ImplementationGraph`` (that is, the specific ``ImplementationGraph``
```

Comment:
I wonder if there's something clearer than "highest level" here, which suggests abstraction. My first thought was "root" but it's not really arranged like a tree.

Just spitballing here, but another idea would be to call the pipeline-size versions `StepGraph` and `ImplementationGraph`, and call these (which can be partial) `StepGraphFragment` and `ImplementationGraphFragment` or similar.

Reply:
I'm not sure I understand your suggestion. We already have a unique name for the pipeline-size versions of `StepGraph` and `ImplementationGraph` - `PipelineSchema` and `PipelineGraph`.

I see your point about "highest level" though. What about simply "largest"?

Reply:
Largest is good!